jobrunner/local_run.py

```diff
@@ -77,6 +77,11 @@ def add_arguments(parser):
         help="Don't stop on first failed action",
         action="store_true",
     )
+    parser.add_argument(
+        "--timestamps",
+        help="Emit timestamps on log out. Useful in CI",
```

Comment:
Nitpick, but the primary audience for the help text here are researchers looking at `opensafely run --help`. So I think the CI comment is just extra noise and potential confusion here. "Log" is also slightly confusing here as there are separate log file which always include timestamps. So I'd just go for:
```
Include timestamps in output
```
The CI thing could go in a Python comment above. And actually, would be worth doing the same for `--continue-on-error`.

Reply:
Good call. There's already such a comment on the `--continue-on-error` definition :)